src/.nuget/Microsoft.NETCore.Runtime.CoreCLR/win/Microsoft.NETCore.Runtime.CoreCLR.pkgproj

```diff
@@ -9,19 +9,20 @@
     <PackageTargetRuntime>$(MinOSForArch)-$(PackagePlatform)</PackageTargetRuntime>
   </PropertyGroup>
   <ItemGroup>
-    <ArchitectureSpecificNativeFile Include="$(BinDir)clretwrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)coreclr.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)dbgshim.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordaccore.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)clretwrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)coreclr.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)dbgshim.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordaccore.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscordbi.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.debug.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)mscorrc.dll" />
+    <ArchitectureSpecificNativeFileAndSymbol Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)mscorlib.ni.dll" />
     <ArchitectureSpecificNativeFile Include="$(BinDir)System.Private.CoreLib.ni.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.debug.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)mscorrc.dll" />
-    <ArchitectureSpecificNativeFile Include="$(BinDir)sos.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)System.Private.CoreLib.dll" />
     <ArchitectureSpecificLibFile Include="$(BinDir)mscorlib.dll" />
```

Comment:
Hadn't thought about them not being adjacent in the symbol package... I tried this out in a test feed, and it looks like having them adjacent is necessary for the "symbol match" green light to turn on, but it doesn't seem affect the pdb being available from the symbol server. (I can download it using symchk on the dll without the pdb adjacent in the symbol package.) I'd say there's nothing to worry about.